lib/cdo/script_constants.rb

```diff
@@ -0,0 +1,54 @@
+# Info on existing tutorial scripts on Code Studio,
+# sometimes referenced in Pegasus.
+# Used for conditional behaviors.
+
+module ScriptConstants
+  HOC_2013_NAME = 'Hour of Code' # this is the old (2013) hour of code
```

Comment:
Please avoid refactorings that increase the scope of this PR until after
HOC.

On Tue, Dec 1, 2015 at 9:52 AM, Mehal Shah <notifications@github.com> wrote:

> In lib/cdo/script_constants.rb
> <https://github.com/code-dot-org/code-dot-org/pull/5712#discussion_r46312354>
> :
>
> > @@ -0,0 +1,54 @@
> > +# Info on existing tutorial scripts on Code Studio,
> > +# sometimes referenced in Pegasus.
> > +# Used for conditional behaviors.
> > +
> > +module ScriptConstants
> > +  HOC_2013_NAME = 'Hour of Code' # this is the old (2013) hour of code
>
> I feel like there's a way to make this an enum, so we don't have to extend
> tests every time we write something new
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/5712/files#r46312354>.
>